WinUIGallery/ControlPages/AppBarPage.xaml

```diff
@@ -1,4 +1,4 @@
-﻿<!--
+<!--
```

Comment:
I don't think users can actually visit this page anymore.

Reply:
Removed!